src/mono/mono/mini/driver.c

```diff
@@ -2325,12 +2339,21 @@ mono_main (int argc, char* argv[])
 			mname = argv [++i];
 			mono_graph_options = MONO_GRAPH_CFG;
 			action = DO_DRAW;
+#ifndef ENABLE_NETCORE
 		} else if (strcmp (argv [i], "--debug") == 0) {
 			enable_debugging = TRUE;
+#endif
```

Comment:
I think we should still support passing `--debug` in netcore mode even though it is a no-op, just for compatibilities sake (e.g. having a MONO_ENV_OPTIONS set)

Reply:
I still like `-nodebug` or maybe `-optimize=nodebug` but so verbose.